includes/api/ApiQueryLogEvents.php

```diff
@@ -156,7 +156,7 @@ public function execute() {
 
 		// SUS-2779
 		$userIds = $this->getFieldFromResults( $res, 'log_user' );
-		$userNames = $this->getIndexedUserNames( $userIds );
+		$userNames = User::whoAre( $userIds );
 
 		foreach ( $res as $row ) {
 			if ( ++ $count > $limit ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this increment or decrement operator into a dedicated statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS881)